django/test/client.py

```diff
@@ -421,93 +425,126 @@ def _get_path(self, parsed):
         # Refs comment in `get_bytes_from_wsgi()`.
         return path.decode("iso-8859-1")
 
-    def get(self, path, data=None, secure=False, **extra):
+    @staticmethod
+    def _transform_headers(headers):
+        return {
+            HttpHeaders.to_header_name(header_name): value
+            for header_name, value in headers.items()
+        }
```

Comment:
method only used in one place - inline it

(also, generally I try to avoid staticmethod in Python - see [this post](https://www.seporaitis.net/posts/2020/05/05/python-staticmethod-usage/) for some reasons)